### PR TITLE
Delete subjects

### PR DESCRIPTION
**Pull request by naid**

### CODE REVIEW

File: resources/views/supervisor/subjects/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                                <div class="form-group">
+                                    {{ Form::label('description', trans('message.description')) }}
+                                    {!! Form::textarea('description', $subject->description, [
+                                        'required' => 'required',
+                                        'placeholder' => trans('message.subject_description_placeholder'),
+                                        'class' => 'form-control'])
+                                    !!}
+                                </div>
+                                <div class="form-group">
+                                    {!! Form::submit('Update', ['class' => 'btn btn-success']) !!}
```
Comment (dinhhoanglong91) [inline resources/views/supervisor/subjects/edit.blade.php]: trans

Comment (naid) [inline resources/views/supervisor/subjects/edit.blade.php]: this is now fixed in the updated PR. Thanks.

**Comment by nguyenthanhtung88** [comment]
OK

**Comment by wataridori** [comment]
OK

**Pull request merged**